community.md

```diff
@@ -0,0 +1,60 @@
+# Content for Node.js Community Webpage
+
+## Introduction
+
+Node.js is an open source software built, used, and maintained by the community. We believe community to be at the core of NodeJS' popularity. Amazing folks from different parts of the world, with a common interest, collaborate and shape this community.
+
+In its earliest times, when Node.js was still in its initial releases, it became extremely popular. Thousands of developers were using it. With the increasing usage, it became difficult for the Node.js core team to keep the software up to date with the user requirements. So thus formed the Node.js foundation in 2015.
+
+Node.js foundation is an open source community, and anyone can become a part of it by contributing to its projects. Open source doesn’t always have the best reputation for being friendly and welcoming, and that makes us sad. Everyone belongs in open source, and Node.js foundation is dedicated to making you feel welcome.
```

Comment:
1. After the first line, add a line, `Community outreach is an initiative under the umbrella of Node.js foundation to connect with the Node.js community.` 
2. In the second line, `Open source doesn't ...`, start it with, `We know, open source ...`.